modules/iam/groups/locals.tf

```diff
@@ -0,0 +1,129 @@
+locals {
+  allow_manage_own_credentials = jsonencode({
```

Comment:
Adding here a comment explaining why we need such policy would be good as well, you can attach link to this article: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_aws_my-sec-creds-self-manage.html

Reply:
Added in [216c662](https://github.com/nvminhtue/tyson_devops_ic/pull/23/commits/216c6628bd63c3a616b795c11beae32c41478579)